### DISCUSSION TITLE
Security Onion Not Detecting

### DISCUSSION THREAD

**Discussion by elamigosam** [Security-Onion-Solutions/securityonion#7625] [Unsupported Versions] [answered]
I am testing the Security Onion 2 software,
got the ISO, burned to a usb and installed on hardware (core i5 32gb Ram 1tb SSD)
I went through the Security Onion Essentials video playlist on Youtube and followed the video instructions to install and configure it. 
I am on the last video of the play list, and decide to test it by running Nmap from a VM. 
Security Onion is not detecting my nmap (nmap 192.168.2.0/24) scan.
"ET POLICY Spotify P2P Client" is the only alert i gotten so far from a 30 computers network. 

I was trying to verify the network configuration from the documentation site:
https://docs.securityonion.net/en/16.04/network-configuration.html
but this documentation references a ubuntu install, and the default ISO install is on Centos(???) 
I am not too familiar with centos configuration files. 
is there a way to run the network configuration wizard to make sure I have my network configuration correct? 

thanks

**Comment by tomjones1977** [comment, marked as answer]
What does your network laydown look like?  Have you got a network tap and/or configured a SPAN port on your switch?

**Reply by elamigosam** [reply]
Currently I have it connected to a switch where everything connect to, before the router. 
I dont think i have a network tap or a SPAN port, are this required?
I am new to cyber security tools, but i need a way to detect if anything bad is going on my network.

**Reply by tomjones1977** [reply]
The reason you're not currently seeing much is due to the way that switches work.  By default SecurityOnion is only going to see traffic that is either broadcast or multicast, or destined for SO itself.

In order to view network traffic from your other devices you need to configure the switch to mirror that traffic to your SO server, this is usually done with a SPAN port, adn the steps will be different depending on the make and model of switch.  Some switches are not capable of configuring a SPAN port - if this is the case you will need to either purchase another switch or configure your network differently.

**Reply by elamigosam** [reply]
oh, ok
this helps. 
i wish this was mentioned in the video series. 

Ok, i will look into this.

thank you.

**Reply by dougburks** [reply]
It's mentioned in the documentation:
https://docs.securityonion.net/en/2.3/hardware.html#packets
